src/main/java/graphql/execution/instrumentation/parameters/InstrumentationExecuteOperationParameters.java

```diff
@@ -12,48 +12,13 @@
 @PublicApi
 public class InstrumentationExecuteOperationParameters {
     private final ExecutionContext executionContext;
-    private final InstrumentationState instrumentationState;
-
     public InstrumentationExecuteOperationParameters(ExecutionContext executionContext) {
-        this(executionContext, executionContext.getInstrumentationState());
-    }
-
-    private InstrumentationExecuteOperationParameters(ExecutionContext executionContext, InstrumentationState instrumentationState) {
         this.executionContext = executionContext;
-        this.instrumentationState = instrumentationState;
     }
 
-    /**
```

Comment:
since the state is now passed into the methods and the old methods removed - these old state extraction methods are no longer needed.